pkg/apis/troubleshoot/v1beta2/collector_shared.go

```diff
@@ -31,8 +31,10 @@ type Secret struct {
 }
 
 type LogLimits struct {
-	MaxAge   string `json:"maxAge,omitempty" yaml:"maxAge,omitempty"`
-	MaxLines int64  `json:"maxLines,omitempty" yaml:"maxLines,omitempty"`
+	MaxAge    string `json:"maxAge,omitempty" yaml:"maxAge,omitempty"`
+	MaxLines  int64  `json:"maxLines,omitempty" yaml:"maxLines,omitempty"`
+	SinceTime string
```

Comment:
why not add json and yaml tags for these two fields too?

Reply:
I think it would be pointless to make since-time part of the specs, because I can't imagine a vendor fixing a date and time for the logs, and the user shouldn't modify the specs either. And in the case of since, because it overrides maxAge, so it would be redundant to add it to the specs. I could add them though.

Reply:
Makes sense, no need to add them, just wanted to make sure we're excluding them intentionally.